### PR TITLE
[MM-45117] Most active dms

### PR DESCRIPTION
**Pull request by BenCookie95**
#### Summary
Adding most active dms for `My` insights.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-45117

#### Screenshots
Screen Shot 2022-07-28 at 3 46 49 PM

#### Release Note

```release-note
Adding most active dms for my insights
```

### CODE REVIEW

File: components/activity_and_insights/insights/top_dms_and_new_members/top_dms_and_new_members.test.tsx
Comment (plant99) [inline components/activity_and_insights/insights/top_dms_and_new_members/top_dms_and_new_members.test.tsx:46]: Not sure if it's necessary, can we also add `outgoing_message_count` to this testdata?

File: components/activity_and_insights/insights/top_dms_and_new_members/top_dms_item/top_dms_item.tsx
Comment (plant99) [inline components/activity_and_insights/insights/top_dms_and_new_members/top_dms_item/top_dms_item.tsx:59]: I was curious about the behaviour if the second user doesn't exist in a team? Should I do this check in backend?

**Review by mkraft — APPROVED** [review]

File: components/activity_and_insights/insights/top_dms_and_new_members/top_dms_item/top_dms_item.tsx
Comment (BenCookie95) [reply components/activity_and_insights/insights/top_dms_and_new_members/top_dms_item/top_dms_item.tsx:59]: That's a good question, if the user is not part of the team should they appear in the top dms in that team? Or are the DMs team agnostic?

cc @matthewbirtch

Comment (matthewbirtch) [reply components/activity_and_insights/insights/top_dms_and_new_members/top_dms_item/top_dms_item.tsx:59]: DMs are team agnostic anyway, so I would say we don't factor in the team at all in to this widget

**Review by plant99 — APPROVED** [review]

**Comment by matthewbirtch** [comment]
@BenCookie95 am I able to test this in a cloud test server or is there another test server I can test?

**Comment by BenCookie95** [comment]
> @BenCookie95 am I able to test this in a cloud test server or is there another test server I can test?

@matthewbirtch I think I'll need to setup a different test server because the server branch has a different name! I'll send it to you in a bit

**Review by matthewbirtch — APPROVED** [review]
Awesome work @BenCookie95. As discussed, the truncation and the modal hover state needs to be fixed and you mentioned you'll do that before merging. I'm going to approve to unblock this

**Comment by BenCookie95** [comment]
@furqanmlk Would need to test locally with this branch https://github.com/mattermost/mattermost-server/pull/20690

**Review by furqanmlk — COMMENTED** [review]
### 1. Profiles icons are not aligned when adding position

image

### 2. Model `Sent` column should be `Messages sent` and `Received` column should be `Messages received`

image

### 3. Model window is getting out

image

**Comment by BenCookie95** [comment]
> ### 1. Profiles icons are not aligned when adding position
> image
> 
> ### 2. Model `Sent` column should be `Messages sent` and `Received` column should be `Messages received`
> image
> 
> ### 3. Model window is getting out
> image
@furqanmlk 
1. Fixed
2. Matt and I decided to change this to `sent` and `received`, he just forgot to update the design
3. Already have a ticket for that, out of curiosity though, are you purposely playing around with your window height?

**Comment by furqanmlk** [comment]
> @furqanmlk
> 
> 1. Fixed
> 2. Matt and I decided to change this to `sent` and `received`, he just forgot to update the design
> 3. Already have a ticket for that, out of curiosity though, are you purposely playing around with your window height?

Thanks @BenCookie95 
For 3. If I have Developer mode window open (i.e Network or Application etc) Model window behaves like that.

**Review by furqanmlk — APPROVED** [review]
Thanks @BenCookie95 

Working as expected

**Pull request merged**
